includes/admin/class.llms.admin.user.custom.fields.php

```diff
@@ -358,6 +402,17 @@ public function save( $user ) {
 
 		}
 
+		// Save personal options.
+		if ( user_can( $user, 'edit_courses' ) ) {
+			if ( 'create' === $action ) {
+				$autosave = 'yes';
+			} else {
+				$autosave = empty( $_POST['llms_builder_autosave'] ) ? 'no' : 'yes';
+			}
+			update_user_meta( $user->ID, 'llms_builder_autosave', $autosave );
+
+		}
```

Comment:
```suggestion
		// Save personal options.
		if ( user_can( $user, 'edit_courses' ) && 'create' !== $action ) {
			$autosave = empty( $_POST['llms_builder_autosave'] ) ? 'no' : 'yes';
			update_user_meta( $user->ID, 'llms_builder_autosave', $autosave );
		}
```
That's because when the 'llms_builder_autosave' user option is empty, we assume it's "yes":
https://github.com/gocodebox/lifterlms/blob/cd71c4b469f5de7983e39457649125e0f4de21cf/includes/admin/class.llms.admin.builder.php#L82-L83

what do you think?